bin/test-2.3/actionmailer

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+rbenv local 2.3.8 &&
+  bundle install --gemfile Gemfile.2.3 &&
+  pushd actionmailer &&
+  bundle exec --gemfile ../Gemfile.2.3 rake test ; popd
+  
```

Comment:
It looks like the github linter isn't recognizing the new line at the end of the file for some reason. Maybe it's context specific since the ruby files are working correctly?

Reply:
that's because there are two space on that line

Reply:
Doh! I see them, I'll update to remove